client/src/components/index.js

```diff
@@ -0,0 +1,4 @@
+import { Title, Button, Logo } from './UI';
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
I saw this comment line two times maybe you should here to add this rule in our eslint config